daemon/tests/harness/mod.rs

```diff
@@ -134,6 +147,29 @@ impl Maker {
             .unwrap()
             .unwrap();
     }
+
+    pub async fn accept_rollover(&self, order: Order) {
+        self.cfd_actor_addr
+            .send(CfdAction::AcceptRollOver { order_id: order.id })
+            .await
+            .unwrap()
+            .unwrap();
+    }
+
+    pub async fn get_update_cfd_proposal(
+        &mut self,
+        order_id: &OrderId,
+    ) -> anyhow::Result<UpdateCfdProposal> {
```

Comment:
No need to qualify `Result`.

```suggestion
    ) -> Result<UpdateCfdProposal> {
```